packages/larva-patterns/components/c-button/c-button.prototype.js

```diff
@@ -6,5 +6,7 @@ module.exports = {
 	c_button_text: 'Sign Up',
 	c_button_screen_reader_text: 'Sign Up',
 	c_button_url: '',
+	c_button_tracking_label: '',
```

Comment:
What do you think about these have a naming convention so that it is clear they are not to be used for other data, and only as parameters? e.g. `c_button_tracking_label_param`, or something like that?

Reply:
@laras126  Is it possible to have a complex variable like this?

```
c_button_ga_tracking: { 
   label: '',
   data: '',
},
```

Because the label is the name of the tracking to be explicit aligned with ga tracking eventLabel

Then we can use tag is like this 
```
{{ wp_action( 'pmc_do_render_custom_ga_tracking', c_button_ga_tracking ) }}
```